### PR TITLE
Provide backwards compatibility for MetaStoreUtils for Coral

### PR DESCRIPTION
**Pull request by laurachenyu**
Build hive-apache for coral integration in https://github.com/prestosql/presto/pull/4661.

### CODE REVIEW

File: src/main/java/org/apache/hadoop/hive/metastore/MetaStoreUtils.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package org.apache.hadoop.hive.metastore;
+
+/**
+ * Coral depends on a different version of Hive, which has the MetaStoreUtils to a different location. This change
+ * allows Coral to work across Hive versions.
+ * Ref: PRESTO-2013
```
Comment (phd3) [inline src/main/java/org/apache/hadoop/hive/metastore/MetaStoreUtils.java]: nit: we can remove the ticket number :) It'd be good to add any useful details from there to the commit message.

Comment (electrum) [reply src/main/java/org/apache/hadoop/hive/metastore/MetaStoreUtils.java]: Agreed, I was going to ask about this.

File: src/main/java/org/apache/hadoop/hive/metastore/MetaStoreUtils.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+ */
+public class MetaStoreUtils extends HiveMetaStoreUtils  {
+}
```
Comment (electrum) [inline src/main/java/org/apache/hadoop/hive/metastore/MetaStoreUtils.java]: Please format like
```java
public class MetaStoreUtils extends HiveMetaStoreUtils {}
```
Also remove extra space before opening `{`

**Review by electrum — APPROVED** [review]

**Comment by electrum** [comment]
Thanks!

**Comment by electrum** [comment]
This is now released as `3.0.0-5`

**Pull request merged**